test-framework/junit5/src/main/java/io/quarkus/test/junit/AppMakerHelper.java

```diff
@@ -0,0 +1,299 @@
+package io.quarkus.test.junit;
+
+import static io.quarkus.test.common.PathTestHelper.getAppClassLocationForTestLocation;
+import static io.quarkus.test.common.PathTestHelper.getTestClassesLocation;
+import static io.quarkus.test.common.PathTestHelper.validateTestDir;
+
+import java.io.IOException;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.InvocationTargetException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+
+import jakarta.enterprise.inject.Alternative;
+
+import org.jboss.jandex.Index;
+
+import io.quarkus.bootstrap.BootstrapConstants;
+import io.quarkus.bootstrap.BootstrapException;
+import io.quarkus.bootstrap.app.AugmentAction;
+import io.quarkus.bootstrap.app.CuratedApplication;
+import io.quarkus.bootstrap.app.QuarkusBootstrap;
+import io.quarkus.bootstrap.app.StartupAction;
+import io.quarkus.bootstrap.model.ApplicationModel;
+import io.quarkus.bootstrap.resolver.AppModelResolverException;
+import io.quarkus.bootstrap.runner.Timing;
+import io.quarkus.bootstrap.utils.BuildToolHelper;
+import io.quarkus.bootstrap.workspace.ArtifactSources;
+import io.quarkus.bootstrap.workspace.SourceDir;
+import io.quarkus.bootstrap.workspace.WorkspaceModule;
+import io.quarkus.commons.classloading.ClassLoaderHelper;
+import io.quarkus.paths.PathList;
+import io.quarkus.runtime.LaunchMode;
+import io.quarkus.test.common.PathTestHelper;
+import io.quarkus.test.common.RestorableSystemProperties;
+import io.quarkus.test.common.TestClassIndexer;
+
+public class AppMakerHelper {
+
+    // TODO Copied from superclass of thing we copied
+    protected static final String TEST_LOCATION = "test-location";
+    protected static final String TEST_CLASS = "test-class";
+    protected static final String TEST_PROFILE = "test-profile";
+    /// end copied
+
+    private static Class<?> quarkusTestMethodContextClass;
+    private static boolean hasPerTestResources;
+
+    private static List<Object> testMethodInvokers;
+    private Runnable configCleanup;
+
+    public static class PrepareResult {
+        protected final AugmentAction augmentAction;
+        public final QuarkusTestProfile profileInstance;
+        protected final CuratedApplication curatedApplication;
+        public final Path testClassLocation;
```

Comment:
This variable doesn't seem to be used besides the ctor init. You are looking for the test class location in `createAugmentor`

Reply:
Ah, sorry, it's initializing `PrepareResult` but still it doesn't appear to be used?

Reply:
You're right, will get rid of it.